tutorials/postgres_truth.ipynb

```diff
@@ -12,9 +12,9 @@
    "metadata": {},
    "source": [
     "Owner: **Joanne Bogart [@jrbogart](https://github.com/LSSTDESC/DC2-analysis/issues/new?body=@jrbogart)**  \n",
-    "Last Verified to Run: **2020-08-03**\n",
+    "Last Verified to Run: **2020-11-24**\n",
     "\n",
-    "This notebook demonstrates access to truth catalog data via the PostgreSQL database at NERSC.  Currently (May 1, 2020) two kinds of truth catalogs are available: star truth and supernova truth.  This notebook will concentrate on star truth. The minion observation database is also available \n",
+    "This notebook demonstrates access to truth catalog data via the PostgreSQL database at NERSC.  Currently (May 1, 2020) two kinds of truth catalogs are available: star truth and supernova truth.  This notebook will concentrate on star truth. The minion observation database is also available, as well as the Run2.2i dr6 object catalog (dpdd columns approximating specification in LSE-163 only).\n",
```

Comment:
Add a link to LSE-163? Also, note that the version we used is 2018 (I just noticed that LSE-163 has an update in 2020)

Reply:
Good idea!

Reply:
Link added.